docs/data/pages.ts

```diff
@@ -119,6 +119,7 @@ const pages: MuiPage[] = [
         pathname: '/x/react-data-grid/recipes',
         children: [
           { pathname: '/x/react-data-grid/recipes-editing', title: 'Editing' },
+          { pathname: '/x/react-data-grid/recipes-styling', title: 'Styling' },
```

Comment:
How about adding it to https://mui.com/x/react-data-grid/style/#styling-cells?
Or converting the "Style" page into a group with a separate "Recipes" page?

<img width="265" src="https://github.com/mui/mui-x/assets/13808724/df21148b-ad01-4408-acc7-01a116156f77">